src/config/dark-sites.config

```diff
@@ -78,6 +78,7 @@ audiolove.club
 audiomass.co
 audioz.download
 augmentedsteam.com
+avatars.alphacoders.com
```

Comment:
This site's homepage isn't dark:
![image](https://user-images.githubusercontent.com/25481501/166149945-538a8405-4ff4-4ae0-ba9b-1c96dd582d90.png)

Reply:
I'm not sure about adding a site because it's broken with Dark Reader, that's not the way to go.